dist/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Infozona</title>
+</head>
+<body>
+    <div id="app"></div>
+    <script src="./bundle.js"></script>
```

Comment:
This is dynamically added by `webpack-html-template`, that is the usual way that the plugin injects script tags. Problem here is that it is pushed to origin in the first place and I'm not sure why since it `dist` folder is added to `.gitignore`.